website/public/js/controllers/groupsCtrl.js

```diff
@@ -173,6 +173,7 @@ habitrpg.controller("GroupsCtrl", ['$scope', '$rootScope', 'Shared', 'Groups', '
           member.petCount = Shared.countPets($rootScope.countExists(member.items.pets), member.items.pets);
           member.mountCount = Shared.countMounts($rootScope.countExists(member.items.mounts), member.items.mounts);
         $scope.profile = member;
+        $scope.e = member.items.gear.equipped;
```

Comment:
I am very against one letter variables. If we're doing this, the member modal jade file should also be worked on in this PR.